.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-  - "0.4"
   - "0.6"
-  - "0.7"
   - "0.8"
```

Comment:
We can live with just 0.10 and 4. 5 is good for catching node bugs early, but is not Long-Term-Support track.